force-app/main/default/classes/TagAnythingBrowserController.cls

```diff
@@ -0,0 +1,11 @@
+public with sharing class TagAnythingBrowserController {
+
+    @AuraEnabled(Cacheable=true)
+    public static List<Tag_Anything__c> requireTags(String search_key){
+        List<list<SObject>> search_tags = new List<list<SObject>>();
```

Comment:
@MariaApAb `List<list<SObject>>` fix to `List<List<SObject>>` in declaration and constructor.